cmd/jetstream-controller/main.go

```diff
@@ -57,6 +57,7 @@ func run() error {
 	cert := flag.String("tlscert", "", "NATS TLS public certificate")
 	key := flag.String("tlskey", "", "NATS TLS private key")
 	ca := flag.String("tlsca", "", "NATS TLS certificate authority chain")
+	tlsfirst := flag.Bool("tlsfirst", false, "If true, then NATS connections will be initiated with a TLS negotiation, rather than using a plain text / STARTTLS approach")
```

Comment:
I would not call it `STARTTLS`, as it has some differences. It's more of `implicit TLS` vs `explicit TLS`. 

Let's maybe write something along te lines "If enabled, forces explicit TLS without waiting for Server INFO".